gap/display.gi

```diff
@@ -202,3 +202,81 @@ if not IsBound(Splash) then  # This function is written by A. Egri-Nagy
     return;
   end);
 fi;
+
+# CR's code
+
+InstallMethod(DotPartialOrderDigraph, "for a partial order digraph",
+[IsDigraph],
+function(graph)
```

Comment:
"graph" -> "digraph" for consistency with other methods, I think. Maybe this is not consistent...